ansible/group_vars/all

```diff
@@ -1078,7 +1166,7 @@ mongodb_users:
 }
 
 # Nex database credentials
-nex_database_host: "{{ rds.endpoint }}"
+nex_database_host: "{{ nex.rds.endpoint }}"
```

Comment:
can we add some guarding - something along the lines of:
`nex.rds.endpoint || rds.endpoint`

so as to keep things backward compatible